pytest.ini

```diff
@@ -2,3 +2,4 @@
 markers =
     online: marks tests that use online resources (deselect with '-m "not online"')
     slow: marks tests that are slow (deselect with '-m "not slow"')
+python_classes =
```

Comment:
Not sure we need this. By default it will look for `test` and we are using `test/`.

Reply:
I have this because the `TestProcess` class starting with `Test` is making it so that `pytest` looks inside it to look for tests, yielding warnings because it has an `__init__` constructor.